test/acceptance/client.js

```diff
@@ -612,27 +654,15 @@ describe('The node.js OC client', function(){
         it('should contain the error details', function(){
 
           var expectedRequestWithExtraParams = {
-            url: 'http://localhost:1234',
-            method: 'post',
+            url: 'http://localhost:1234/hello-world/~1.0.0??hi=john',
```

Comment:
Sorry, not sure I understand why there are 2 ?. What's wrong with 1?
Or, what about http://localhost:1234/hello-world/~1.0.0/?hi=john